public/apps/account/account-nav-button.tsx

```diff
@@ -0,0 +1,109 @@
+/*
+ *   Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+import {
+  EuiContextMenuItem,
+  EuiContextMenuPanel,
+  EuiHeaderSectionItemButton,
+  EuiIcon,
+  EuiPopover,
+  EuiButtonEmpty,
+  EuiAvatar,
+  EuiHorizontalRule,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiTitle,
+  EuiText,
+  EuiListGroupItem,
+  EuiListGroup,
+} from '@elastic/eui';
+import React, { useState } from 'react';
+
+export function AccountNavButton(props: {
+  isInternalUser: boolean;
+  username: string;
+  tenant?: string;
+}) {
+  const [isPopoverOpen, setPopoverOpen] = useState<boolean>(false);
+  const horizontalRule = <EuiHorizontalRule margin="xs" />;
+  const username = props.username;
```

Comment:
Minor: camel case

Reply:
I think username is now mostly one word instead of two.